standalone/src/DelphesMain.h

```diff
@@ -3,7 +3,11 @@
 #include "k4SimDelphes/DelphesEDM4HepOutputConfiguration.h"
 
 #include "podio/Frame.h"
+#if PODIO_VERSION_MAJOR > 0 || (PODIO_VERSION_MAJOR == 0 && PODIO_VERSION_MINOR >= 99)
```

Comment:
```suggestion
#if PODIO_BUILD_VERSION >= PODIO_VERSION(0, 99, 0)
```

We have macros that do all of the work :)